boards/nxp/frdm_ke15z/frdm_ke15z.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: frdm_ke15z
+name: NXP FRDM-KE15Z
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+flash: 256
+ram: 32
```

Comment:
`sram_u` is selected as the SRAM on this part, so the ram value should be 24, right?